src/main/resources/org/sagebionetworks/web/client/widget/entity/menu/v2/ActionMenuWidgetViewImpl.ui.xml

```diff
@@ -1,89 +1,95 @@
-				<a:ActionMenuItem icon="THUMB_TACK" iconFixedWidth="true" action="PIN_THREAD">Pin Thread</a:ActionMenuItem> <!-- text and icon updated by listener -->
-				<a:ActionMenuItem icon="TIMES_CIRCLE" iconFixedWidth="true" action="DELETE_THREAD">Delete Thread</a:ActionMenuItem> <!-- text and icon updated by listener -->
-				<a:ActionMenuItem iconFixedWidth="true" action="RESTORE_THREAD">Restore Thread</a:ActionMenuItem> <!-- text and icon updated by listener -->
-				<a:ActionMenuItem icon="PLUS_SQUARE_O" iconFixedWidth="true" action="CREATE_EXTERNAL_DOCKER_REPO">Add External Repository</a:ActionMenuItem>
-				<b:Divider ui:field="actDivider" visible="false" />
-				<b:DropDownHeader ui:field="actHeader">ACT Tools</b:DropDownHeader>
-				<a:ActionMenuItem icon="FLAG" iconFixedWidth="true" action="APPROVE_USER_ACCESS">Change User Access</a:ActionMenuItem>
-				<a:ActionMenuItem icon="FLAG" iconFixedWidth="true" action="MANAGE_ACCESS_REQUIREMENTS">Manage Access Requirements</a:ActionMenuItem>
-			</b:DropDownMenu>
-		</b:DropDown>
-		<bh:Div ui:field="controllerContainer" addStyleNames="displayInlineBlock"/>
-	</g:FlowPanel>
+<!DOCTYPE ui:UiBinder SYSTEM "http://dl.google.com/gwt/DTD/xhtml.ent">
+<ui:UiBinder xmlns:ui="urn:ui:com.google.gwt.uibinder"
+	xmlns:g="urn:import:com.google.gwt.user.client.ui" xmlns:c="urn:import:com.google.gwt.user.cellview.client"
+	xmlns:b="urn:import:org.gwtbootstrap3.client.ui" xmlns:t="urn:import:org.sagebionetworks.web.client.widget.table.v2"
+	xmlns:a="urn:import:org.sagebionetworks.web.client.widget.entity.menu.v2"
+	xmlns:bg="urn:import:org.gwtbootstrap3.client.ui.gwt" xmlns:bh="urn:import:org.gwtbootstrap3.client.ui.html">
+
+	<g:FlowPanel>
+		<b:Tooltip title="Show advanced search">
+			<a:ActionButton visible="false" icon="FILTER" addStyleNames="margin-right-5" action="SHOW_ADVANCED_SEARCH" />
+		</b:Tooltip>
+		<b:Tooltip title="Show simple search">
+			<a:ActionButton visible="false" icon="FILTER" addStyleNames="margin-right-5" action="SHOW_SIMPLE_SEARCH" />
+		</b:Tooltip>
+		<b:Tooltip title="Show query">
+			<a:ActionButton visible="false" icon="DATABASE" addStyleNames="margin-right-5" action="SHOW_QUERY" />
+		</b:Tooltip>
+		<b:Tooltip title="Edit query results">
+			<a:ActionButton visible="false" icon="EDIT" addStyleNames="margin-right-5" action="EDIT_TABLE_DATA" />
+		</b:Tooltip>
+		<b:DropDown addStyleNames="displayInlineBlock">
+			<b:Button dataToggle="DROPDOWN" type="PRIMARY" text="Download Options" marginRight="5" visible="false" ui:field="tableDownloadOptions"/>
+			<b:DropDownMenu addStyleNames="dropdown-menu-right">
+				<a:ActionMenuItem action="DOWNLOAD_TABLE_QUERY_RESULTS">Export Table</a:ActionMenuItem>
+                <b:Tooltip ui:field="addToDownloadListMenuItemTooltip" placement="LEFT" container="body"
+                           text="A draft version of a Dataset cannot be added to the Download List." >
```

Comment:
Is this tooltip shown when the user does not have edit permission on the Dataset?